src/AeroSharp/DataAccess/Internal/MapOperations.cs

```diff
@@ -54,4 +54,13 @@ IMapEntryGenerator mapEntryGenerator
     /// <returns> The map operation. </returns>
     public static Operation RemoveByKey<TKey>(string bin, TKey key, IMapEntryGenerator mapEntryGenerator) =>
         MapOperation.RemoveByKey(bin, mapEntryGenerator.GenerateKey(key), MapReturnType.KEY_VALUE);
+
+    /// <summary>
+    ///     Get the key and value rank for a map
+    /// </summary>
+    /// <param name="bin"> The bin containing the map. </param>
+    /// <param name="rank"> The rank to fetch. </param>
+    /// <returns> The map operation. </returns>
+    public static Operation GetByRank(string bin, int rank) =>
+        MapOperation.GetByRank(bin, rank, MapReturnType.KEY_VALUE);
```

Comment:
One major thing here that is an assumption as the rest of this file, should we just default (unsettable by customers) that the return type of the function should always be a KV?

Reply:
I think so. If we want, we could also provide additional methods like `GetValuesByRank` or `GetKeysByValueRank` to provide different access patterns. I personally think keeping things uniform and simple for now is the way to go, until someone has a specific need for one of these other return types.

Reply:
Sounds good to me!  Let the people raise issues!